src/utils/editor/index.js

```diff
@@ -64,6 +64,10 @@ export function shouldShowPrettyPrint(selectedSource) {
     return false;
   }
 
+  if (isLoading(selectedSource)) {
+    return true;
+  }
```

Comment:
shouldnt we also check if the source is pretty?

Reply:
Now that we are showing some loading, we should do it for any source while it's loading?